object-store/1.1/modules/ROOT/pages/object-store-connector-reference.adoc

```diff
@@ -345,6 +346,82 @@ If `Expiration Interval` is set to a value lower than or equal to zero, then the
 If not provided, the Mule runtime's default ObjectStoreManager is used. Setting this parameter is only necessary when you want to use a non-default Object Store implementation. For example, if you want to have a store which is backed by Redis or JDBC. |  |
 |===
 
+
+
+[[objectStorev2]]
+=== Object Store (Object Store v2)
+
+
+NOTE: These settings apply only to xref:object-store::index.adoc[Object Store v2].
+
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Persistent a| Boolean | Whether the store is persistent or transient. | true |
+| Max Entries a| | *Not configurable for Object Store v2* 
+
+The maximum number of entries is unlimited.|  |
+| Entry Ttl a| Number a| The entry timeout.
+
+Ignored if the value of `expirationInterval` is less than or equal to `0`.
+
+* Minimum value: 1 second
+* Maximum value: 30 days
+
+If the field contains: 
+
+* No value
++
+** Mule version 4.2.x, 4.3.0, and later: Rolling TTL
+** Mule versions earlier than 4.2.1: Static TTL of 30 days by default
+
+* Less than or equal to `0`
++
+TTL set to the maximum value (2592000 seconds/30 days).
+* Positive number
++
+TTL set to the specified value
+
+// ** Mule version 4.2.x, 4.3.0, and later: TTL set to the specified value and creates a rolling TTL. 
```

Comment:
I realized that 4.2.x is not correct. Because there is 4.2.0 - that we never touched.

So should be >=4.2.1, 4.3.x. Not sure what is the best to write that

Reply:
4.2.1 and later